src/tasks/block.rs

```diff
@@ -161,6 +173,27 @@ impl BlockBuilder {
         self.bundle_poller.evict();
     }
 
+    async fn filter_transactions(&mut self, in_progress: &mut InProgressBlock) {
```

Comment:
does self need to be `&mut` here? can it not be `&self` ?

Reply:
https://github.com/init4tech/builder/pull/31/commits/349c96e4767781d50c0c7ca48ac26b28b06311bf